docs/blog/connected-registry.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Connected Registry Private Preview
+description: Private preview for ACR connected registry feature.
+ms.topic: post
+ms.date: 01/05/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Private Preview - ACR Connected Registry Feature
+We are announcing the private preview of Azure Container Registry (ACR) connected registry feature.
+
+The connected registry feature of ACR allows you to deploy a registry on your premises and synchronize images between the ACR and your premises. It brings the container images and OCI artifacts closer to your on-premise container workloads and increases their performance and reliability.
+
+ACR connected registry can be used in conjunction with Azure IoT Edge, Azure Arc, Azure Stack as well as other edge container workloads.
```

Comment:
Can we make IoT Edge and Azure Stack product links?
Is "other edge container" workloads the right terminology? I thought "edge" was verboten? @dlepow?

Reply:
Edge (lowercase) is an industry term so I would assume we are OK to use it. @dlepow let me know if this should be changed.

Reply:
I think(?) "edge" is OK. We use it in a lot of product names :)

Reply:
Added links to the IoT Edge and Azure Stack marketing pages @SteveLasker.